frame/crowdloan-rewards/src/lib.rs

```diff
@@ -238,14 +240,14 @@ pub mod pallet {
 		/// ```haskell
 		/// proof = sign (concat prefix (hex reward_account))
 		/// ```
-		#[pallet::weight((<T as Config>::WeightInfo::associate(TotalContributors::<T>::get()), Pays::No))]
+		#[pallet::weight(<T as Config>::WeightInfo::associate(TotalContributors::<T>::get()))]
 		#[transactional]
 		pub fn associate(
 			origin: OriginFor<T>,
 			reward_account: T::AccountId,
 			proof: ProofOf<T>,
 		) -> DispatchResultWithPostInfo {
-			T::AssociationOrigin::ensure_origin(origin)?;
+			ensure_none(origin)?;
```

Comment:
Are regular users able to provide Origin::None? Shouldn't this just be ensure_signed, but not using the account_id provided?

Reply:
`ensure_none` just ensure that the extrinsic has not been signed from what the doc is describing. I've seen that you can send unsigned extrinsic using polkadotjs

Reply:
[It looks like it is the intended behavior](https://github.com/paritytech/polkadot/blob/master/runtime/common/src/claims.rs#L317)